docs/content/envoy-primer.md

```diff
@@ -207,6 +210,7 @@ When Envoy receives a policy decision, it expects a JSON object with the followi
 * `request_headers_to_remove` (optional): is an array of string header names
 * `http_status` (optional): a number representing the HTTP status code
 * `body` (optional): the response body
+* `dynamic_metadata` (optional): an object to be added to the `CheckResponse`'s `DynamicMetadata` field.
```

Comment:
`an object representing dynamic metadata to be consumed by the next Envoy filter.`